lcmgen/emit_cpp.c

```diff
@@ -128,6 +128,7 @@ map_type_name(const char *t)
 
 void setup_cpp_options(getopt_t *gopt)
 {
+    getopt_add_string (gopt, 0, "cpp-std",    "c++98",      "C++ standard(c++98, c++11)");
```

Comment:
Please add a check that the value passed in for --cpp-std is valid. emit_cpp() is a good place to put that check. This will help catch typos - e.g., if someone accidentally invokes lcm-gen with '--cpp-std=c+11'